alert-policies/activemq/MessageEnqueueRate.yml

```diff
@@ -0,0 +1,27 @@
+name: Message Eequeue Rate
```

Comment:
Please check the name. It should be "Message Enqueue Rate"

Reply:
Hi @pkosanam,
Review changes are done.